File: Extra_Long_Factorials.js
'use strict';

process.stdin.resume();
process.stdin.setEncoding('utf-8');

let inputString = '';
let currentLine = 0;

process.stdin.on('data', function(inputStdin) {
    inputString += inputStdin;
});

process.stdin.on('end', function() {
    inputString = inputString.split('\n');

    main();
});

function readLine() {
    return inputString[currentLine++];
}

/*
 * Complete the 'extraLongFactorials' function below.
 *
 * The function accepts INTEGER n as parameter.
 */

function extraLongFactorials(n) {
  let result=BigInt(1);
  for(let i=BigInt(`${n}`); i>1; i--)
    {
      result=result*i;
    }
  console.log(result.toString().replace('n',"")) ;
};

function main() {
    const n = parseInt(readLine().trim(), 10);

    extraLongFactorials(n);
}

File: Extra_Long_Factorials.py
#!/bin/python3

import math
import os
import random
import re
import sys

#
# Complete the 'extraLongFactorials' function below.
#
# The function accepts INTEGER n as parameter.
#

def extraLongFactorials(n):
    ausgabe = n
    for q in range(1, n):
        ausgabe *= (n - q)
    return print(ausgabe)

if __name__ == '__main__':
    n = int(input().strip())

    extraLongFactorials(n)
